### PR TITLE
Upgrade to Kotlin 1.3.40

### PR DESCRIPTION
**Pull request by lavruk**
Also in the request:
- added iosArm32 for older devices 
- updated coroutines to 1.2.2 (compatible with 1.3.40)
- updated stately to 0.7.3 (compatible with 1.3.40)

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -22,61 +22,63 @@ repositories {
 kotlin {
     targets {
         jvm()
-        iosX64()
+        iosX64("native")
```
Comment (benasher44) [inline build.gradle.kts]: Is there a way we can leave this without a label? It then feels weird down below referencing the "native" target, but really it's an iOS target

Comment (benasher44) [reply build.gradle.kts]: I think we can create a separate native source set that isn't bound to a particular target and the continue using the dependsOn pattern below

File: build.gradle.kts [hunk 2]
```diff
@@ -22,61 +22,63 @@ repositories {
...
                 implementation("org.jetbrains.kotlin:kotlin-test-common")
                 implementation("org.jetbrains.kotlin:kotlin-test-annotations-common")
-                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-common:${coroutinesVersion}")
-                implementation("co.touchlab:stately:${statelyVersion}")
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-common:$coroutinesVersion")
+                implementation("co.touchlab:stately:$statelyVersion")
             }
         }
 
-         val jvmMain by getting {
```
Comment (benasher44) [inline build.gradle.kts]: Minor nit: extra space

File: gradle.properties [hunk 3]
```diff
@@ -1 +1 @@
-VERSION=0.1.0
\ No newline at end of file
+VERSION=0.1.1
```
Comment (benasher44) [inline gradle.properties]: I think this deserves a 0.2.0, no?

File: build.gradle.kts (hunk 1 above)
Comment (lavruk) [reply build.gradle.kts]: Sure, it makes sense

File: gradle.properties (hunk 3 above)
Comment (lavruk) [reply gradle.properties]: It had no API changes, so I thought bumping the `patch` is enough.

**Comment by lavruk** [comment]
I'm wondering why would Stately 0.7.2 has windows artifact and 0.7.3 does not, did they remove it?

**Comment by benasher44** [comment]
No I checked with @kpgalligan. Windows update should be coming soon

**Comment by benasher44** [comment]
Was hoping closing and opening the PR would kick CI

**Comment by benasher44** [comment]
Thanks @lavruk! I'm unfortunately out of town this week, but I'll get this published first thing next week.

**Pull request merged**